.github/workflows/main.yml

```diff
@@ -23,4 +23,17 @@ jobs:
         done
         for file in $files; do
           miniwdl check $file
-        done
\ No newline at end of file
+        done
+  
+  import_syntax_check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Check import syntax
+      run: |
+        files=$(find . -name '*.wdl')
+        bad_lines=$(awk '/import/' $files | awk '!/master/')
```

Comment:
I think you probably want to search for the full URL string (not just `master`), just to be safe.